pulsar-discovery-service/src/main/java/org/apache/pulsar/discovery/service/server/ServiceConfig.java

```diff
@@ -27,6 +27,8 @@
 
 import com.google.common.collect.Sets;
 
+import io.netty.handler.ssl.ClientAuth;
```

Comment:
I don't think we should reuse the Netty enum as it is. We are shading Netty and that will prevent users to pass the correct type. 
Also, if we really have 2 valid values (Optional and Required, since None won't make much sense), we should just use a boolean.

Reply:
Sorry, had forgotten to remove unused imports - Rookie mistake.